Assignment1/Assignment1_QNo9/SolAssignmentNo1_QNo9.rb

```diff
@@ -0,0 +1,14 @@
+# => 9. How to convert 'ruby' to :ruby ?
+# => author : Hrisikesh Singha Chowdhury
+
+# => Fn to implement buisness logic
+def cvt_to_sym
+  # => Repeateadly asking users to give correct input strings
+  puts "Enter string"
+  str = gets.chomp
+  puts "The required symbolic name is #{str.to_hash.value}"
```

Comment:
@HrisikeshSinghaChowdhury There is a inbuilt method to_sym in ruby , please use that.